decryptor/postgresql/data_encoder_test.go

```diff
@@ -160,8 +160,19 @@ func TestTextMode(t *testing.T) {
 			logMessage: `Can't decode int value and no default value`},
 
 		// encryption/decryption integer data, not tokenization
-		{input: []byte("some data"), decodedData: []byte("some data"), encodedData: []byte(strDefaultValue), decodeErr: nil, encodeErr: nil,
-			setting: &config.BasicColumnEncryptionSetting{Tokenized: false, DataType: "int32", ResponseOnFail: "default", DefaultDataValue: &strDefaultValue}},
+		{
+			input:       []byte("some data"),
+			decodedData: []byte("some data"),
+			encodedData: []byte(strDefaultValue),
+			decodeErr:   nil,
+			encodeErr:   nil,
+			setting: &config.BasicColumnEncryptionSetting{
+				Tokenized:        false,
+				DataType:         "int32",
+				ResponseOnFail:   "default_value",
```

Comment:
Hard coded values are more readable, imho. Good tests should be readable - because review from the person is the only way to verify correctness of the tests. 
But on the other hand, use of constants makes introducing changes (renames) easily. So, it makes sense. I'll do it